### DISCUSSION TITLE
Hide Team Radio and Pop Out Window

### DISCUSSION THREAD

**Discussion by JJWatMyself** [f1multiviewer/issue-tracker#285] [Feature requests] [answered]
Similar to RCM, it would be great if Team Radio could have an option to be hidden from the main Live Timing window and added as a pop-out window.

![image](https://user-images.githubusercontent.com/22590445/201363451-c0fe1b85-1272-4b3b-b766-33def96bf6ab.png)

On the pop-out list as vertical list instead of horizontal
![image](https://user-images.githubusercontent.com/22590445/201363772-0af147e8-9176-446e-b575-48081c2c049d.png)

Potential to add option to auto play new messages also.

**Comment by jeppeolesen** [comment, marked as answer]
This has been added and can either be set in the MultiViewer settings for all sessions, or individually under the live timing settings for each session.

![Screenshot 2025-03-23 at 01 35 09](https://github.com/user-attachments/assets/bdd42efb-70da-4026-a27b-8c1eb0b27c32)

![Screenshot 2025-03-23 at 01 35 23](https://github.com/user-attachments/assets/8eac28dc-c02c-45f9-9c7a-050344015e07)
